Cargo.toml

```diff
@@ -122,6 +122,9 @@ tracing-subscriber = { version = "=0.3.18", features = ["env-filter", "json"] }
 typomania = { version = "=0.1.2", default-features = false }
 url = "=2.5.4"
 unicode-xid = "=0.2.6"
+async-compression = { version = "=0.4.18", default-features = false, features = ["gzip", "tokio"] }
+krata-tokio-tar = "=0.4.2"
+tokio-util = { version = "=0.7.12", features = ["compat"] }
```

Comment:
can you add them with alphabetic order? 🙏

Reply:
sure!